recipes/hazelcast-cpp-client/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class HazelcastCxx(ConanFile):
+    name = "hazelcast-cpp-client"
+    description = "C++ client library for Hazelcast in-memory database."
+    license = "Apache-2.0"
+    topics = ("conan", "hazelcast", "client", "database", "cache")
+    homepage = "https://github.com/hazelcast/hazelcast-cpp-client"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_openssl": False
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def validate(self):
+        if self.settings.os == "Linux" and self.settings.compiler.libcxx!="libstdc++11":
+            raise ConanInvalidConfiguration("Requires settings.compiler.libcxx = libstdc++11")
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def requirements(self):
+        self.requires("boost/1.75.0")
+        if self.options.with_openssl:
+            self.requires("openssl")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+        # This small hack might be useful to guarantee proper /MT /MD linkage
+        # in MSVC if the packaged project doesn't have variables to set it
+        # properly
+        tools.replace_in_file(str(self._source_subfolder) + "/CMakeLists.txt", "LANGUAGES CXX)",
+                              '''LANGUAGES CXX)
+include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
+conan_basic_setup()''')
```

Comment:
replace in files must be done it `build()` (in CCI we need to support lots of workflows)

With my suggestion above you should be able to remove this 

```suggestion
```